GettingStarted/FindingDocs.ipynb

```diff
@@ -0,0 +1,267 @@
+    "<br>Verified Stack Release: **16.0**\n",
+    "\n",
+    "### Learning Objectives:\n",
+    "\n",
+    "In this notebook we will look at a few different ways to find the documentation on a given DM Stack function or class. \n",
+    "After working through this tutorial you should be able to: \n",
+    "1. Use the jupyter notebook built-in functions to read the docstrings of Stack classes and functions \n",
+    "2. Use the `whereis` Stack Club utility to locate DM Stack web documentation.\n",
+    "\n",
+    "### Logistics\n",
+    "This notebook is intended to be runnable on `lsst-lspdev.ncsa.illinois.edu` from a local git clone of https://github.com/LSSTScienceCollaborations/StackClub.\n",
+    "\n",
+    "## Set-up\n",
+    "We'll need the `stackclub` package to be installed. If you are not developing this package, you can install it using `pip`, like this:\n",
+    "```\n",
+    "pip install git+git://github.com/LSSTScienceCollaborations/StackClub.git#egg=stackclub\n",
+    "```\n",
+    "If you are developing the `stackclub` package (eg by adding modules to it to support the Stack Club tutorial that you are writing, you'll need to make a local, editable installation. In the top level folder of the `StackClub` repo, do:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! cd .. && python setup.py -q develop --user && cd -"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When editing the `stackclub` package files, we want the latest version to be imported when we re-run the import command. To enable this, we need the %autoreload magic command."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inline Notebook Help\n",
+    "\n",
+    "Command line tasks have usage information - try running them with no arguments, or `--help`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! imageDifference.py --help"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The pipeline task python code also contains useful docstrings, accessible in various ways:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from lsst.pipe.tasks.imageDifference import ImageDifferenceTask"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "help(ImageDifferenceTask)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "help(ImageDifferenceTask.run)"
```

Comment:
Using weekly `w_2018_31` I get
```
AttributeError: type object 'ImageDifferenceTask' has no attribute 'run'
```

I'll try this on v16.0 and report back.

Reply:
`help(ImageDifferenceTask.run)` works in v16.0 but not in `w_2018_31` which illustrates that this is changing. See https://community.lsst.org/t/api-change-for-tasks-rename-run-primarymethod-to-rundataref-run/3054 on how this is being renamed to `runDataRef`.

Is there a different example that could be substituted?

Reply:
Good idea - I'll find one. Thanks!